### PR TITLE
javasript week1 homework first commit

### PR DESCRIPTION
**Pull request by Lokesh-sw**

### CODE REVIEW

File: javascript/javascript1/week1/1.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+// Age-IFy excericse
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:1]: It is good to have a habit to give a meaningful name to files, I wouldn't say 1.js is the right approach regarding naming

File: javascript/javascript1/week1/1.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+const yearOfBirth = 1987;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log(
+  "You will be" + " " + age + " " + "years old in" + " " + yearFuture + "."
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:8]: Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: javascript/javascript1/week1/1.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
+// Age-IFy excericse
+
+const yearOfBirth = 1987;
+let yearFuture = 2027;
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:4]: why let?

File: javascript/javascript1/week1/1.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+const age = yearFuture - yearOfBirth;
+
+console.log(
+  "You will be" + " " + age + " " + "years old in" + " " + yearFuture + "."
+);
+
+// Goodboy-oldboy excercise
+
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:12]: I would use Prettier 🙂

File: javascript/javascript1/week1/1.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+const dogYearOfBirth = 2017;
+
+let dogYearFuture = 2027;
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:15]: why let?

File: javascript/javascript1/week1/1.js [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+const dogYear = dogYearFuture - dogYearOfBirth;
+
+let shouldShowResultInDogYears = dogYear;
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:19]: `shouldShowResultInDogYearsWe` should be boolean (either true or false), and here it is 10
The assignment says: _add an extra variable called shouldShowResultInDogYears. If it is true we should show the result in dog years, if it is false we should show it in human years._

File: javascript/javascript1/week1/1.js [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+      " " +
+      "human years old in" +
+      " " +
+      dogYearFuture
+  );
+}
+
+//Housey pricey
+
+let volumeInMeters = [8 * 10 * 10, 5 * 11 * 8];
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:45]: why let? 🙂  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

File: javascript/javascript1/week1/1.js [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+let gardenSizeInM2 = [100, 70];
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:46]: why let, it should be const?

File: javascript/javascript1/week1/1.js [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+const houseCost = [2500000, 1000000];
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:47]: 👍

File: javascript/javascript1/week1/1.js [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+  "Services",
+  "Advisory",
+  "Enterprise",
+  "Portfolio",
+  "Administration",
+  "Inc",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
+const startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:109]: What if you need two random numbers per array? 🙂

File: javascript/javascript1/week1/1.js [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+  "The startup:" +
+    " " +
+    startupName +
+    " " +
+    "contains" +
+    " " +
+    startupName.length +
+    " " +
+    "characters"
+);
```
Comment (shravanipagilla) [inline javascript/javascript1/week1/1.js:121]: Hej  Lokesh, you did very  hard work  . i felt nothing to change .

**Review by shravanipagilla — COMMENTED** [review]
nice work

File: javascript/javascript1/week1/1.js [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+
+if (estimatedCostJulia < houseCost[1]) {
+  console.log(
+    "Julia is paying" + " " + houseCost[1] - estimatedCostJulia + " " + "extra"
+  );
+} else if (estimatedCostJulia > houseCost[1]) {
+  console.log("Julia saving" + " " + estimatedCostJulia - houseCost[1]);
+} else if (estimatedCostJulia === houseCost[1]) {
+  console.log("Correct Price");
+}
```
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:79]: Have you checked if this code is working? 
It doesn't btw. 
I will paste the corrected version of your homework here but you should be aware that you shouldn't submit the homework that doesn't work 🙂 
```
// for Peter
const estimatedCostPeter = volumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300;
if (estimatedCostPeter < houseCost[0]) {
  console.log(`Peter is paying ${houseCost[0] - estimatedCostPeter} extra`);
} else if (estimatedCostPeter > houseCost[0]) {
  console.log(`Peter saving ${estimatedCostPeter - houseCost[0]}`);
} else {
  console.log("Correct Price");
}

// for Julia
const estimatedCostJulia = volumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300;
if (estimatedCostJulia < houseCost[1]) {
  console.log(`Julia is paying ${houseCost[1] - estimatedCostJulia} extra`);
} else if (estimatedCostJulia > houseCost[1]) {
  console.log(`Julia saving ${estimatedCostJulia - houseCost[1]}`);
} else {
  console.log("Correct Price");
}
```

**Review by cecastosic — COMMENTED** [review]
@Lokesh-sw Please go through my comments and try to understand when you should use `let` and when `const`. Maybe this video can help https://youtu.be/1mgLWu69ijU 
Be careful before submitting the homework, the submitted code should work. Thanks 🌸

File: javascript/javascript1/week1/week1.js [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+const houseCost = [2500000, 1000000];
+
+//for peter
+
+const estimatedCostPeter =
+  volumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300;
+if (estimatedCostPeter < houseCost[0]) {
+  console.log(`Peter is paying ${houseCost[0] - estimatedCostPeter} extra`);
+} else if (estimatedCostPeter > houseCost[0]) {
+  console.log(`Peter saving ${estimatedCostPeter - houseCost[0]}`);
```
Comment (cecastosic) [inline javascript/javascript1/week1/week1.js:42]:
```suggestion
  console.log(`Peter is saving ${estimatedCostPeter - houseCost[0]}`);
```

File: javascript/javascript1/week1/week1.js [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+// for julia
+
+const estimatedCostJulia =
+  volumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300;
+if (estimatedCostJulia < houseCost[1]) {
+  console.log(`Julia is paying ${houseCost[1] - estimatedCostJulia} extra`);
+} else if (estimatedCostJulia > houseCost[1]) {
+  console.log(`Julia saving ${estimatedCostJulia - houseCost[1]}`);
```
Comment (cecastosic) [inline javascript/javascript1/week1/week1.js:54]:
```suggestion
  console.log(`Julia is saving ${estimatedCostJulia - houseCost[1]}`);
```

Comment (cecastosic) [reply javascript/javascript1/week1/week1.js:54]: You can also notice that you are repeating some of the words ('is saying', 'is paying', 'extra'...)

**Review by cecastosic — APPROVED** [review]
Cool! 🚀 But please remove 1.js file before merging 🙂 and change HTML so it uses the new js file.

File: javascript/javascript1/week1/index.html [hunk 15]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>smart-ease-io</title>
+  </head>
+  <body>
+    <script src="1.js"></script>
```
Comment (cecastosic) [inline javascript/javascript1/week1/index.html]: You should also change here which file you want to use.

**Pull request merged**